backend/matching/data/data_generator.py

```diff
@@ -1,7 +1,11 @@
 import argparse
 import sys
-from llsc.backend.matching.data.seeder.data_seeder import Seeder
-from config import OUTPUT_FORMAT_CHOICES, FILE_PATH_REQUIRED_FORMATS
+from backend.matching.data.seeder.data_seeder import Seeder
+from backend.matching.data.config import (
+    OUTPUT_FORMAT_CHOICES,
+    OPTIONS_FOR_DATA,
+    FILE_PATH_REQUIRED_FORMATS,
+)
 
 
 class CLI:
```

Comment:
make this a pdm script command so we can access `faker` and `pandas` packages. You can add this in `backend/pyproject.toml` under `[tool.pdm.scripts]` as `matching: python3 matching/data/data_generator`